contracts/PoaNetworkConsensus.sol

```diff
@@ -62,12 +62,15 @@ contract PoaNetworkConsensus is IPoaNetworkConsensus {
         _;
     }
 
-    function PoaNetworkConsensus(address _masterOfCeremony) public {
+    function PoaNetworkConsensus(address _masterOfCeremony, address[] validators) public {
         // TODO: When you deploy this contract, make sure you hardcode items below
         // Make sure you have those addresses defined in spec.json
         require(_masterOfCeremony != address(0));
         masterOfCeremony = _masterOfCeremony;
         currentValidators = [masterOfCeremony];
+        for (uint256 y = 0; y < validators.length; y++) {
+            currentValidators.push(validators[y]);
```

Comment:
We don't check here, that `validators[y] != masterOfCeremony`. Should we?

Reply:
MoC is already added to list of validators, we assume that people should not double include a validator. Worse case, they will re deploy the contract if it is deployed incorrectly

Reply:
Ok, seems it is ready to merge in this case.